Beams/MRML/vtkMRMLRTBeamNode.h

```diff
@@ -248,7 +308,16 @@ class VTK_SLICER_BEAMS_MODULE_MRML_EXPORT vtkMRMLRTBeamNode : public vtkMRMLMode
   /// Couch angle
   double CouchAngle;
 
+  /// Dose influence matrix
+  DoseInfluenceMatrixType DoseInfluenceMatrix;
+
+  /// Dose grid dimensions (on which the dose influence matrix is defined)
+  double DoseGridDim[3];

```

Comment:
This should not be double. The get/set functions use int. I'd use unsigned int probably.

Reply:
Also now we can initialize values in the header like:
`unsigned int DoseGridDim {5, 5, 5};`